docker-compose.yml

```diff
@@ -1,10 +1,134 @@
+# This docker-compose file will stand up the entire stack including ingress and inventory.
 version: "3"
 services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=32181
+      - ZOOKEEPER_SERVER_ID=1
+  kafka:
+    image: confluentinc/cp-kafka
+    ports:
+      - 29092:29092
+    depends_on:
+      - zookeeper
+    environment:
+      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:29092
+      - KAFKA_BROKER_ID=1
+      - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:32181
+  puptoo:
+    image: quay.io/cloudservices/insights-puptoo:latest
+    command:
+      - python
+      - -m
+      - puptoo.app
+    ports:
+      - 8000:8000 #for prometheus endpoint
+    environment:
+      - REJECTION_TOPIC=platform.upload.validation
+      - LOGLEVEL=INFO
+    depends_on:
+      - kafka
+  minio:
+    image: minio/minio
+    command: server /data
+    volumes:
+      - ~/mnt/data:/data:z
+      - ~/mnt/config:/root/.minio:z
```

Comment:
It should be possible to run minio without these volumes mounted, like with the postgres service.

Reply:
When createbuckets runs in its own container, doesn't it need to share a volume with this minio container? If you remove the volume mounts from both services and run `d-c up`, visit `localhost:9000`, do you see the buckets created?

Reply:
You are right, the do need to share the volume. I may have run the create bucket commands manually.
Can we try to use named volumes though?

Reply:
Sure, do you mind if we push that off to another PR? I'm trying to get this working for now to fix uploads in CI.

Reply:
👍